spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,151 @@
+require "rails_helper"
+
+RSpec.describe "The Guest Show Page", type: :feature do
+  before(:each) do
+    @hotel = Hotel.create!(name:"Vail Inn", location:"Vail")
+
+    @room1 = @hotel.rooms.create!(rate: 125, suite: "Presidential")
+    @room2 = @hotel.rooms.create!(rate: 130, suite: "Executive")
+    @room3 = @hotel.rooms.create!(rate: 100, suite: "Basic")
+    
+    @guest1 = Guest.create!(name: "Ethan", nights: 3)
+    @guest2 = Guest.create!(name: "Zahava", nights: 3)
+    @guest3 = Guest.create!(name: "Ezzy", nights: 1)
+
+    @room1.guests << @guest1
+    @room1.guests << @guest2
+    
+    @room2.guests << @guest1
+    @room2.guests << @guest3
+
+    @room3.guests << @guest1
+  end
+
+  # =========================
+  # STORY 1: GUEST SHOW TESTS
+  # =========================
+
+  it "I see a guest's name" do
+    visit "/guests/#{@guest1.id}"
+    
+    within("div#guest_info") do
+      expect(page).to have_content(@guest1.name)
+      expect(page).to_not have_content(@guest2.name)
+    end
+    
+    visit "/guests/#{@guest2.id}"
```

Comment:
This is a very thorough feature test- keep this up!

Reply:
The things I see that make it thorough: 
* Using within blocks
* Using `.to` and `.to_not` lines inside within blocks
* Doing all of that on multiple pages